internal/analytics/tilt_analytics_test.go

```diff
@@ -0,0 +1,223 @@
+package analytics
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/windmilleng/wmclient/pkg/analytics"
+)
+
+type testCase struct {
+	name         string
+	opt          analytics.Opt
+	expectRecord bool
+}
+
+func testCases(expectedWhenOptedIn, expectedWhenOptedOut, expectedWhenNoOpt bool) []testCase {
+	return []testCase{
+		{"opt in", analytics.OptIn, expectedWhenOptedIn},
+		{"opt out", analytics.OptOut, expectedWhenOptedOut},
+		{"opt default", analytics.OptDefault, expectedWhenNoOpt},
+	}
+}
+
+type optSetting struct {
+	opt analytics.Opt
+}
+
+func (os *optSetting) setOpt(opt analytics.Opt) error {
+	os.opt = opt
+	return nil
+}
+
+func TestCount(t *testing.T) {
+	for _, test := range testCases(true, false, false) {
+		t.Run(test.name, func(t *testing.T) {
+			ma := analytics.NewMemoryAnalytics()
+			os := &optSetting{}
+			a := NewTiltAnalytics(test.opt, os.setOpt, ma)
+			a.Count("foo", map[string]string{}, 1)
```

Comment:
why not verify that the tags get sent along, too? (and in incr/timer cases as well)